### DISCUSSION TITLE
Android api<24 not support if_nametoindex method

### DISCUSSION THREAD

**Discussion by Kotlinlife** [markqvist/Reticulum#299] [Help & Questions]
I try wrap reticulum by using chaquo library but it is crash in if_nametoindex function

**Comment by markqvist** [comment]
Since chaquo support is not on the roadmap, or currently a planned feature, and there is not much information provided here, I will just move this to the discussions for now.

If you'd like others to help and/or investigate, feel free to add more info about exactly what you are trying to do, what goes wrong, and how to reproduce.
